### DISCUSSION TITLE
Ability to query all alerts in ngalert api

### DISCUSSION THREAD

**Discussion by harrisondscott** [grafana/grafana#57764] [Alerting]
As an end user of Grafana I would like to be able to query alert rule groups within a folder on the api, this will enable alert querying globally. With this feature enhancement we would be able to see all alerts via the api

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
